### PR TITLE
Notify OnAdLoaded on LoadAd call

### PR DESCRIPTION
**Pull request by dragmz**
Enables OnAdLoaded testing with a dummy Reward Ad client.

### CODE REVIEW

**Comment by dragmz** [comment]
@googlebot I signed it!

File: source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs [hunk 1]
```diff
@@ -52,6 +52,10 @@ public void CreateRewardedAd(string adUnitId)
         public void LoadAd(AdRequest request)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+
```
Comment (stowy) [inline source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs]: please remove whitespace on this line

File: source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs [hunk 2]
```diff
@@ -52,6 +52,10 @@ public void CreateRewardedAd(string adUnitId)
...
+            var onAdLoaded = OnAdLoaded;
```
Comment (stowy) [inline source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs]: no need for this local variable, just use the `OnAdLoaded` variable directly.

File: source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs [hunk 3]
```diff
@@ -52,6 +52,10 @@ public void CreateRewardedAd(string adUnitId)
...
+            if (onAdLoaded != null)
```
Comment (stowy) [inline source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs]: please add { } braces around this if statement as per the conventions in this codebase.

**Pull request merged**